packages/page-generator/src/helpers/getStandardRow.tsx

```diff
@@ -0,0 +1,59 @@
+import {
+  BaseComponentPropsWithChildren,
+  getBaseHTMLProps,
+  ScreenSize,
+} from '@norges-domstoler/dds-components';
+import { SectionGeneratorRow } from '../types';
+import { MultiValue, SingleValue } from 'react-select';
+import { ChangeEvent } from 'react';
+import { getComponent } from '.';
+import React from 'react';
+
+type T = HTMLInputElement & Record<string, never>;
+
+const Row = (
+  props: BaseComponentPropsWithChildren<HTMLElement, SectionGeneratorRow>
+) => {
+  const { id, className, htmlProps, ...rest } = props;
+  if (props.as === 'div') {
```

Comment:
`props.as` burde sikkert ha en default verdi siden den er optional, sånn at folk ikke bare får den tomme stien(les: `<></>`) når de glemmer å legge ved en verdi for `as`.

Reply:
Fixed. Returnerer nå Fragment som wrapper om `as` ikke er definert, siden dagens løsning har fragment som wrapper. Så det vil ikke være noen endring for de som bruker den idag.